curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff7f1fa497db1ef87bac1d.md

```diff
@@ -0,0 +1,148 @@
+---
+id: 63ff7f1fa497db1ef87bac1d
+title: Step 10
+challengeType: 0
+dashedName: step-10
+---
+
+# --description--
+
+Inside your `forEach`, destructure the `author`, `image`, `url`, and `bio` items. You don't need to assign them to anything, just put them in a curly brace inside the `forEach`.
+
+Also, put an `index` parameter inside the `forEach`, but it should be outside of the destructuring. This `index` will represent the position of each authors. It'll be useful for fetching just what you want and paginating the authors. 
```

Comment:
Tried to make this a bit more general in some places, and clear up the language a bit:

```suggestion
Pass an empty callback function to the `forEach` method. For the first parameter of the callback, destructure the `author`, `image`, `url`, and `bio` items. You don't need to assign them new names – just wrap those keys in curly braces.

For the second parameter, pass in `index`. This will represent the position of each author, and will be useful for pagination later.
```